### PR TITLE
Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by KenbonTN**
-[] About myself section is aligned with flexbox
-[] I used CSS Grid and Flexbox combined to build the project.
-[]  ul and li are used for categories in the "Work" section. Html, CSS, and JavaScript are on the list of items.
-[] I don't need to implement the detail modal window that requires JavaScript.
-[] I have designed the project sticking to the design in Figma as much as possible.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -73,8 +73,246 @@ <h4 class="connect font-style">LET'S CONNECT</h4>
...
+                <p class="notice font-style">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="programming-languages font-style">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button class="project-button" type="button">See Project</button>
+            </section>
```
Comment (Bondok6) [inline index.html]: - [x] Great job here 💯 However, according to the requirements and your Figma design you should Implement the button interactions (enable, hover, etc.). So kindly add them 👍

image

- [x] Kindly, don't forget to apply it to all your buttons 👍

File: style.css [hunk 2]
```diff
@@ -141,3 +151,447 @@ section {
...
+
+.btn:hover {
+  cursor: pointer;
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.languages {
+  margin-top: 30px;
+}
```
Comment (Bondok6) [inline style.css]: - [x] According to the requirements you must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So I suggest adding some left and right margin or padding to match your Figma design 💯

| Your Design  | Figma Design  |   
| --- | --- |
|   |  |

File: index.html [hunk 3]
```diff
@@ -73,8 +73,246 @@ <h4 class="connect font-style">LET'S CONNECT</h4>
...
         </div>
 
+    </div>
+    <div class="cards grid-container">
+        <section class="first-card card">
+            <img class="snapshoot-style" src="images/Snapshoot Portfolio.svg" alt="Snapshoot portfolio image">
+            <section>
+                <h1 class="card-header font-style">
+                    Tonic
+                </h1>
```
Comment (Bondok6) [inline index.html]: - [OPTIONAL] I suggest removing the `<h1>` tag from here and adding another hx (`<h2>`, `<h3>`, ....) 👌 . to Follow best practices for HTML **Only Use H1 Elements Once Per Page for optimal SEO**. for more info, you can check this [Link](https://techstacker.com/seo-only-use-h1-elements-once-per-page/).

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @KenbonTN 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No Linter errors ✅
- Correct GitHub flow ✅
- Your repo is professional ✅
- Professional README file ✅


## Required Changes ♻️
- [ ] _Check the comments under the review._ ⬇

## Optional suggestions
- [ ] _Check the comments under the review._ ⬇

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 
Hi @KenbonTN ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉🎉

happy


## Highlights 💯 
- No linter errors ✔️ 
- Descriptive commit message ✔️ 
- Descriptive PR ✔️ 
- Professional Readme ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**